### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by krt16s** [Kadabash/SyncOnWifi#2]
````
:app:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/home/krt/src/fdroiddata/build/org.t2.synconwifi/app/src/main/java/org/t2/synconwifi/AccountListAdapter.java:127: error: cannot find symbol
                + mContext.getApplicationContext().getString(R.string.account_image_description)
                                                                     ^
  symbol:   variable account_image_description
  location: class string
1 error
:app:compileReleaseJavaWithJavac FAILED

````

**Comment by Kadabash** [comment]
Oh, that string got lost while translating.
Fixed in commit 98685ea3ba8aca9a817a9088c0103b1816f8fd0c .
Does it build now?

**Comment by krt16s** [comment]
Yep, looks good now. Thanks!

````
INFO: Successfully built version 1.0 of org.t2.synconwifi
DEBUG: > /home/krt/tmp/fdroid/android-sdk-linux/build-tools/25.0.0/aapt dump xmltree build/org.t2.synconwifi/app/build/outputs/apk/app-release-unsigned.apk AndroidManifest.xml
DEBUG: Checking build/org.t2.synconwifi/app/build/outputs/apk/app-release-unsigned.apk
DEBUG: > /home/krt/tmp/fdroid/android-sdk-linux/build-tools/25.0.0/aapt dump badging build/org.t2.synconwifi/app/build/outputs/apk/app-release-unsigned.apk
INFO: success: org.t2.synconwifi
INFO: Finished.
INFO: 1 builds succeeded
````
